interfaces/interfaces.go

```diff
@@ -0,0 +1,15 @@
+package interfaces
+
+//go:generate moq -out mock/mock_redisclienter.go -pkg mock . RedisClienter
+
+import (
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+// RedisClienter - interface for redis
+type RedisClienter interface {
+	Set(string, interface{}, time.Duration) *redis.StatusCmd
```

Comment:
I feel like adhering to good coding practices means these parameters should have names. Without the names someone using this interface has no idea what each of the parameters is

Reply:
Also I am not sure what a `StatusCmd` does but good rule of thumb to follow is return interface types as it doesn't couple you to a specific implementation. Depending on the complexity of the `StatusCmd` struct consider creating your own interface to cover its behaviour and return that instead.

Reply:
Creating my own interface for this broke the Client. Leaving as is for now.